rinja_parser/src/memchr_splitter.rs

```diff
@@ -0,0 +1,97 @@
+pub(crate) trait Splitter: Copy {
+    /// The any of the needles was found in the haystack, then split the haystack at the first hit.
```

Comment:
I think something went wrong with this explanation. XD

Missing a word maybe between `The` and `any` at the beginning?

Reply:
"The" should have been "if". Common typo, the "the" key is right next to the "if" key. 🙈

Reply:
Obviously 😛